service-review/src/main/java/com/iwa/reviewservice/review/Review.java

```diff
@@ -0,0 +1,97 @@
+package com.iwa.reviewservice.review;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+
+@Entity
+public class Review {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private int rating;
```

Comment:
rating should be between 0 and 5

Reply:
add checking if rating is between 0 and 5